lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -660,8 +633,50 @@ class CkParagraphBuilder implements ui.ParagraphBuilder {
     return properties;
   }
 
+  /// Determines if the given [text] contains any code points which are not
+  /// supported by the current set of fonts.
+  void _verifyFontsSupportText(String text) {
```

Comment:
nit: rename to `_ensureFontsSupportText` ("verify" does not imply a side-effect that this method has).

Reply:
Done.